From: Bhavana Nagendra <bnagendr@redhat.com>
Subject: Re: [RHEL5] Disable PCI mmconf and segmentation on HP xw9300/9400
Date: Fri, 27 Oct 2006 09:00:59 -0400
Bugzilla: 219159
Message-Id: <4542030B.8010405@redhat.com>
Changelog: Disable PCI mmconf and segmentation on HP xw9300/9400


Based on conversations between AMD, HP and comments added to BZ 190123,
DL585 G2 needs to be blacklisted as well.    This patch does just that.   
Reviewed and tested by HP.   Please ACK and include in RHEL5B2.

--- linux-2.6.18.x86_64/arch/x86_64/pci/mmconfig.c.orig	2006-10-27 08:57:17.000000000 -0400
+++ linux-2.6.18.x86_64/arch/x86_64/pci/mmconfig.c	2006-10-27 08:59:20.000000000 -0400
@@ -191,6 +191,13 @@ static struct dmi_system_id __devinitdat
 			DMI_MATCH(DMI_PRODUCT_NAME, "HP xw9400 Workstation"),
 		},
 	},
+        {
+                .callback = disable_mmconf,
+                .ident = "ProLiant DL585 G2",
+                .matches = {
+                        DMI_MATCH(DMI_PRODUCT_NAME, "ProLiant DL585 G2"),
+                },
+        },
 
 	{}
 };

